codecov.yml

```diff
@@ -12,18 +12,22 @@ coverage:
         target: auto
         threshold: null
         branches: null
+        informational: true
 
     patch:
       default:
         target: auto
         branches: null
+        informational: true
 
     changes:
       default:
         branches: null
+        informational: true
 
 
 comment:
   layout: "header, diff, changes, sunburst, uncovered, tree"
   branches: null
   behavior: default
+  after_n_builds: 8 # Wait until all 8 of the test suite jacoco test builds have been uploaded before writing a comment to the pr (no more incomplete coverage report emails)
```

Comment:
nice find

Reply:
https://docs.codecov.com/docs/codecovyml-reference#codecovnotifywait_for_ci is another option that isn't tied to the build specifics

Reply:
@lbergelson I'm testing it over in #7818. If it works there I'll pull that commit into this branch and merge.

Reply:
Louis i've tried a few modes of that option. It doesn't seem to work so far as i can tell. There is probably something I misunderstand. I'm going to merge this approach since it does work.